component/src/main/java/org/wso2/carbon/identity/authenticator/pinterest/PinterestAuthenticatorConstants.java

```diff
@@ -63,4 +63,7 @@ private PinterestAuthenticatorConstants() {
 	public static final int CONNECTION_TIMEOUT_VALUE = 15000;
 	//Constant for read time out
 	public static final int READ_TIMEOUT_VALUE = 15000;
+	public static final String AUTHORIZATION = "Authorization";
+	public static final String AUTHENTICATION_BEARER = "Bearer ";
+	public static final String AUTHENTICATION_BASIC = "Basic ";
 }
```

Comment:
And new line at the EOF

Reply:
Fixes: https://github.com/wso2-extensions/identity-outbound-auth-pinterest/pull/14/commits/970babdc86f1f0d5cbd20a7a779424109f6d0a9e